tests/app/views/contexts/summary/test_question.py

```diff
@@ -78,8 +78,10 @@ def test_create_question(self):
             question_schema,
             answer_store=self.answer_store,
             schema=self.schema,
-            list_item_id=None,
             rule_evaluator=self.get_rule_evaluator(),
+            location=None,
+            block_id="",
```

Comment:
These should be realistic, as block ID would never not exist at this point. Same for the rest.